frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -675,3 +675,142 @@ export const GRAPH_AXIS_SETTINGS = {
   // DEPRECATED" replaced with "label" series setting
   "graph.series_labels": {},
 };
+
+export const SANKEY_DATA_SETTINGS = {
+  ...columnSettings({
+    getColumns: (
+      [
+        {
+          data: { cols },
+        },
+      ],
+      settings,
+    ) => cols,
+    hidden: true,
+  }),
+  "graph._start_filter": {
```

Comment:
Since these settings are only applicable to Sankey chart, we use the chart name as a prefix so we don't confuse these settings with more generic ones.

e.g. https://github.com/metabase/metabase/blob/a48893b44db7905f5e1165e84a0a0032992938f9/frontend/src/metabase/visualizations/visualizations/WaterfallChart.jsx#L24-L29

Reply:
They've been renamed.